src/main/kotlin/pt/up/fe/ni/website/backend/util/FileUploader.kt

```diff
@@ -0,0 +1,47 @@
+package pt.up.fe.ni.website.backend.util
+
+import com.cloudinary.Cloudinary
+import com.cloudinary.Transformation
+import java.io.File
+
+interface FileUploader {
+    fun upload(folder: String, fileName: String, image: ByteArray): String
```

Comment:
```suggestion
    fun uploadFile(folder: String, fileName: String, image: ByteArray): String
```
This is specific for images (since we resize them). There will probably be a regular upload in the future